spec/integration/directly_contains_one_association_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'spec_helper'
+
+describe ActiveFedora::Base do
+  before do
```

Comment:
directly_contains breaks when you use ActiveFedora::Base instead of ActiveFedora::File.  I'll submit an issue and a PR with tests to show the problem.